### PR TITLE
Lit3311 

### PR DESCRIPTION
**Pull request by nafisa-valieva**
Here I have updated LIT 3311, (added a list of witnesses, an abstract, incipit and explicit.). UBTaeth28 has not been merged, but I hope it is ok!

### CODE REVIEW

**Review by karljonaskarlsson — APPROVED** [review]

File: 3001-4000/LIT3311Prosph.xml [hunk 1]
```diff
@@ -50,12 +64,17 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <term key="Aks"/>
             </keywords>
          </textClass>
+         <abstract>
+            <p>A letter (prosphoneticus) by <persName ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek, discusses the nature of Christ, aiming to limit the influence of <persName ref="PRS7553Nestoriu"/>.
```
Comment (MassimoVilla) [inline 3001-4000/LIT3311Prosph.xml:68]: `<foreign xml:lang="la">prosphoneticus</foreign>` ?

Comment (nafisa-valieva) [reply 3001-4000/LIT3311Prosph.xml:68]: done!

**Review by MassimoVilla — APPROVED** [review]

File: 3001-4000/LIT3311Prosph.xml [hunk 2]
```diff
@@ -65,7 +65,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
             </keywords>
          </textClass>
          <abstract>
-            <p>A letter (prosphoneticus) by <persName ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek, discusses the nature of Christ, aiming to limit the influence of <persName ref="PRS7553Nestoriu"/>.
+            <p>A letter (<foreign xml:lang="lat">prosphoneticus</foreign>) by <persName ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek, discusses the nature of Christ, aiming to limit the influence of <persName ref="PRS7553Nestoriu"/>.
```
Comment (MassimoVilla) [inline 3001-4000/LIT3311Prosph.xml:68]: Is it `xml:lang="la"` or `xml:lang="lat"`? If it's `lat`, then I should update quite a number of BAV manuscripts, thank you @nafisa-valieva

File: 3001-4000/LIT3311Prosph.xml [hunk 3]
```diff
@@ -30,6 +30,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                   <citedRange unit="item">5219</citedRange>
                </bibl>
             </listBibl>
+         <listWit>
+            <witness corresp="BNFabb18" xml:id="A"/>
```
Comment (eu-genia) [inline 3001-4000/LIT3311Prosph.xml:34]: cp. the remarks given here https://github.com/BetaMasaheft/Works/pull/1310/files same applies

File: 3001-4000/LIT3311Prosph.xml (hunk 2 above)
Comment (eu-genia) [reply 3001-4000/LIT3311Prosph.xml:68]: It is `la` https://betamasaheft.eu/Guidelines/?id=languageIntro

Comment (eu-genia) [reply 3001-4000/LIT3311Prosph.xml:68]: @nafisa-valieva the section on `langUsage `should include a list of all language abbreviations used as `xml:lang` element in the record

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by nafisa-valieva** [comment]
@karljonaskarlsson, would you mind to check the abstract and see if you like it this way? (I applied comments you made to Homilies)

File: 3001-4000/LIT3311Prosph.xml [hunk 4]
```diff
@@ -50,12 +79,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
...
             </keywords>
          </textClass>
+         <abstract>
+            <p>A letter (<foreign xml:lang="la">prosphoneticus</foreign> ) by <persName
+                  ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of
+                  <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek
+               according to <bibl type="CPG">
+                  <ptr target="bm:CPG3"/>
+               </bibl>discusses the nature of Christ, aiming to limit the influence of <persName
+                  ref="PRS7553Nestoriu"/>. Its Gǝʿǝz version is known as part of <title
```
Comment (karljonaskarlsson) [inline 3001-4000/LIT3311Prosph.xml:89]: maybe "transmitted" instead of "known"?

File: 3001-4000/LIT3311Prosph.xml [hunk 5]
```diff
@@ -50,12 +79,26 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                <term key="Aks"/>
             </keywords>
          </textClass>
+         <abstract>
+            <p>A letter (<foreign xml:lang="la">prosphoneticus</foreign> ) by <persName
+                  ref="PRS3255CyrilI"/> addressed to Princesses Arcadia and Marina, sisters of
+                  <persName ref="PRS9480Theodosi"/>. The letter, originally composed in Greek
```
Comment (karljonaskarlsson) [inline 3001-4000/LIT3311Prosph.xml:85]: Yes, I think this looks good! CPG talks specifically about the Geez version, correct?

Maybe one could then also add CPG to the bibliography below (under the category `<listBibl type="secondary">`)?

**Review by karljonaskarlsson — APPROVED** [review]

File: 3001-4000/LIT3311Prosph.xml (hunk 5 above)
Comment (eu-genia) [reply 3001-4000/LIT3311Prosph.xml:85]: If we have the same text in CPG, with a number, this should have a `listBibl type="clavis`" for mapping

Comment (eu-genia) [reply 3001-4000/LIT3311Prosph.xml:85]: https://betamasaheft.eu/Guidelines/?id=work-teiHeader example 4

File: 3001-4000/LIT3311Prosph.xml [hunk 6]
```diff
@@ -125,7 +125,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <ptr target="bm:Bausi2010EAEQerellos"/>
                </bibl>
             </listBibl>
-
+            <listBibl type="secondary">
+               <bibl type="CPG">
+                  <ptr target="bm:CPG3"/>
+                  <citedRange unit="item">5219</citedRange>
```
Comment (eu-genia) [inline 3001-4000/LIT3311Prosph.xml:131]: https://betamasaheft.eu/Guidelines/?id=work-teiHeader check Example 4

Comment (nafisa-valieva) [reply 3001-4000/LIT3311Prosph.xml:131]: changed.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**